src/pages/{content.relativePath}.js

```diff
@@ -276,6 +278,10 @@ export const query = graphql`
             imgRatio
             fullHeight
             imgPlaceholder
+            # iconOverlay {
+            #   icon
+            #   ariaLabel
+            # }
```

Comment:
```suggestion
```

Reply:
We need to uncomment this for the media area to works as intended. I would left the lines here as a reminder.